cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -129,13 +131,16 @@ func (cc *createClusterOptions) createCluster(ctx context.Context) error {
 
 	bootstrapper := bootstrapper.New(&bootstrapperClient{kind, kubectl})
 
+	certgen := crypto.NewCertificateGenerator()
```

Comment:
does generating cert have to be in cmd level?

Reply:
Oh this is just initializing the object for injection along with AwsIamAuth. 
I create the certs in the package level. In AwsIamAuth.go